host/onebox.go

```diff
@@ -200,10 +177,45 @@ type (
 		// LocalTaskWaitTime. defaults to 0ms.
 		LocalTaskWaitTime time.Duration
 
-		// TaskProcessTime. The amount of time spent by the poller in-between requests
-		TaskProcessTime time.Duration
 		// RecordDecisionTaskStartedTime. The amount of time spent by History to complete RecordDecisionTaskStarted
 		RecordDecisionTaskStartedTime time.Duration
+
+		// The pollers that will be created to process
+		Pollers []SimulationPollerConfiguration
+
+		Tasks []SimulationTaskConfiguration
+	}
+
+	SimulationPollerConfiguration struct {
```

Comment:
nonblocking: 

since onebox is used for both docker/local development and simulation, what do you think about pulling this simulation-specific stuff into a simulation.go file so that it's a bit easier to see at a glance if there's any significant changes to the more general onebox config or it's just simulation config?

Reply:
Ack, when I do further refactoring I'll move it out.